### PR TITLE
RavenDB-20484 Client Configuration View: Incorrect value for Load Bal…

### PR DESCRIPTION
**Pull request by kwiato**
…ance Behavior

### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-20484

### Additional description

In addition to fixing the wrong data displayed, fixed the "dbname" in tooltip, stopped the forms from moving when the arrow appears and some minor UX improvements.

### Type of change

- Bug fix

### How risky is the change?

- Not relevant

### Backward compatibility

- Not relevant

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing by Contributor

- It has been verified by manual testing

### Testing by RavenDB QA team

- No special testing by RavenDB QA team is needed

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

**Review by ml054 — APPROVED** [review]

**Comment by ppekrol** [comment]
Please resolve conflicts

**Comment by kwiato** [comment]
Resolved

**Comment by ppekrol** [comment]
Studio tests are failing?

**Comment by ml054** [comment]
@kwiato Please fix studio tests before merge.

File: src/Raven.Studio/typescript/components/pages/database/settings/clientConfiguration/ClientDatabaseConfiguration.tsx [hunk 1]
```diff
@@ -17,9 +29,9 @@ import appUrl = require("common/appUrl");
 import ClientConfigurationUtils from "components/common/clientConfiguration/ClientConfigurationUtils";
 import useClientConfigurationFormController from "components/common/clientConfiguration/useClientConfigurationFormController";
 import { tryHandleSubmit } from "components/utils/common";
-import { PopoverWithHover } from "components/common/PopoverWithHover";
 import useClientConfigurationPopovers from "components/common/clientConfiguration/useClientConfigurationPopovers";
 import { PropSummary, PropSummaryItem, PropSummaryName, PropSummaryValue } from "components/common/PropSummary";
+import classNames = require("classnames");
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/settings/clientConfiguration/ClientDatabaseConfiguration.tsx]: we don't use require, look around. :)

File: src/Raven.Studio/typescript/components/pages/database/tasks/panels/SubscriptionPanel.tsx [hunk 2]
```diff
@@ -180,8 +180,8 @@ function ConnectedClients(props: ConnectedClientsProps) {
             )}
             <div className="connected-clients-items">
                 {connections.Results.length === 0 && <div>No clients connected</div>}
-                {connections.Results.map((connection) => (
-                    <div className="text-center py-2 connected-client">
+                {connections.Results.map((connection, index) => (
+                    <div key={"connectedClient" + index} className="text-center py-2 connected-client">
```
Comment (ml054) [inline src/Raven.Studio/typescript/components/pages/database/tasks/panels/SubscriptionPanel.tsx]: let's use connection.ClientUri here for key generation. 

See: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md

**Review by ml054 — APPROVED** [review]

**Pull request merged**